lib/init.g

```diff
@@ -808,28 +810,30 @@ end );
 ##  then read the first `gaprc' file from the GAP root directories.
 ##
 CallAndInstallPostRestore( function()
-    if READ_GAP_ROOT( "gaprc" ) then
-      GAPInfo.HasReadGAPRC:= true;
-    elif not IsExistingFile(GAPInfo.UserGapRoot) then
-      # For compatibility with GAP 4.4:
-      # If no readable `gaprc' file exists in the GAP root directories and
-      # the user root directory does not (yet) exist then
-      # try to read `~/.gaprc' (on UNIX systems) or `gap.rc' (otherwise).
-      if not ( GAPInfo.CommandLineOptions.r or GAPInfo.HasReadGAPRC ) then
-        if ARCH_IS_UNIX() then
-          GAPInfo.gaprc:= SHALLOW_COPY_OBJ( GAPInfo.UserHome );
-          if IsString(GAPInfo.gaprc) then
-            APPEND_LIST_INTR( GAPInfo.gaprc, "/.gaprc" );
+    if not GAPInfo.CommandLineOptions.ignoreconfig then
```

Comment:
This re-indenation makes it a bit difficult to review the diff. Also, deep nesting is something I personally like to avoid in general.

So, how about changing this patch to just insert these lines:
```gap
  if GAPInfo.CommandLineOptions.ignoreconfig then
    return;
  fi;
```